### ISSUE TITLE
Missing recordings

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/recording-validation-interface#222]
https://speech-db.altlab.app/maskwacis/segment/15446

<img width="762" alt="image" src="https://user-images.githubusercontent.com/15003407/123528232-37da5180-d6a3-11eb-8f22-095053406fc6.png">

**Comment by nienna73** [comment]
The first two recordings are white noise and the second two are rather quiet. I'll look into this and see if it's a pattern with this session.

**Comment by fbanados** [comment]
Also requires linguist analysis for this issue

**Comment by fbanados** [comment]
Session sounds like it's an instance of the problems tracked in #488 .  Will track down session `2016-06-17-PM-DS-_` to confirm, if it is, re-importing with the right `.eaf`/`.wav` combination should fix the issue.

**Comment by fbanados** [comment]
@aarppe Session in server only has one track `.wav` file, for track 004, which does not have a `.eaf` file.  Do you know if there is a backup somewhere for recording sessions from 2016?  That would be the only option to fix this, as otherwise these need to be re-recorded.

**Comment by aarppe** [comment]
Locations where recording audio could have been stored are:
1. One of the two recording MBP laptops.
2. One of the two back-up external drives.
3. "Somewhere" copied over to sapir/whorf (likely in several directories).

As far as I know and recall, a lot of time and effort has been spent to review and rereview the sources above for potentially missing or misplaced audio, and to place these in a unified directory on sapir/whorf. It is still conceivable that some audio might have gotten misplaced or some audio not having copied over properly in that previous process, so that one could still double-check the external drives for missing content. But once that's been done once more and no audio has been found, that should be considered completed.

**Comment by fbanados** [comment]
Yes, I was using `/data/maskwacis-recordings/` as my main source of files, but as I mentioned somewhere in the spreadsheet, it seems that also sessions are backed up at `/data/av/backup-mwe`.  When files are missing in the former, I should check that they are not available in the latter.

**Comment by fbanados** [comment]
I did check the tape backups and could not find the wav files besides track 4 for `2016-06-17-PM-DS-_`, so this issue is on hold waiting for checking backup external drives.

**Comment by fbanados** [comment]
Found the tracks in external drive.

**Comment by fbanados** [comment]
Reimporting fixes the issue.  Whole session should be both re-standardized and validated.
